libraries/stdlib/samples/test/samples/collections/collections.kt

```diff
@@ -311,6 +311,28 @@ class Collections {
             assertPrints(moreFrequencies, "{o=1, t=4, f=2, s=2, e=2, n=1}")
         }
 
+        @Sample
+        fun joinTo() {
+            val sb = StringBuilder("An existing string and a list:")
+            val numbers = listOf(1, 2, 3)
+            assertPrints(numbers.joinTo(sb, prefix = "[", postfix = "]").toString(), "An existing string and a list:[1, 2, 3]")
+            
+            val lotOfNumbers = (1..100).asIterable()
```

Comment:
`IntRange` is already an `Iterable`. What does this conversion intend to show?

Reply:
Yes, it's maybe more clean that way:
```
val lotOfNumbers: Iterable<Int> = 1..100
```